yarn.lock

```diff
@@ -1652,11 +1652,81 @@
     "@uifabric/set-version" "^7.0.23"
     tslib "^1.10.0"
 
+"@fluentui/react-alert@9.0.0-beta.2":
```

Comment:
you should remove the package.json and yarn.lock changes after master is fixed #23634